components/Navigation/index.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import Link from "next/link";
+import Icon from "@mdi/react";
+import { mdiPlusCircle } from "@mdi/js";
+import { mdiBullseyeArrow } from "@mdi/js";
+
+const NavBar = styled.nav`
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0px -1px 0px rgba(58, 72, 80, 0.07),
+    0px -2px 11px -1px rgba(176, 189, 197, 0.12);
+  padding: 15px 0 13px;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  gap: 75px;
+  justify-content: center;
```

Comment:
```suggestion
  justify-content: space-evenly;
```
One line less and looks almost the same, or?